packages/ui/src/components/va-skip-link/VaSkipLink.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { VaSkipLink } from './'
```

Comment:
Please create a docs page in navigation section. Overall any changes should be combined with docs updates.